src/rabbit_msg_store_ets_index.erl

```diff
@@ -68,8 +68,8 @@ delete_object(Obj, State) ->
     true = ets:delete_object(State #state.table, Obj),
     ok.
 
-delete_by_file(File, State) ->
-    MatchHead = #msg_location { file = File, _ = '_' },
+cleanup_undefined_file(State) ->
```

Comment:
Right, then either `clean_up_entries_with_undefined_file` or `clean_up_temporary_reference_count_entries_without_file` is good with me.